lib/network/modules/PhysicsEngine.js

```diff
@@ -296,7 +295,7 @@ class PhysicsEngine {
   /**
    * trigger the stabilized event.
    *
-   * @param {Number} [amountOfIterations=this.stabilizationIterations]
+   * @param {number} [amountOfIterations=this.stabilizationIterations]
```

Comment:
Errmmmm....Looking hard at this. Wasn't expecting a dynamic default. Actually, I think this is legal.